profile/2018-03-01-hybrid.json

```diff
@@ -0,0 +1,156 @@
+{
+    "info":{
+        "name":"2018-03-01-profile",
+        "description":"Profile definition targeted for hybrid applications that could run on azure stack 1807 version and azure cloud"
+    },
+    "resource-manager": {
+        "microsoft.authorization": {
+			"2017-04-01": [
+				"locks"
```

Comment:
There doesn't seem to be a 2017-04-01 version of locks: https://github.com/Azure/azure-rest-api-specs/tree/master/specification/resources/resource-manager